{{cookiecutter.plugin_name}}/.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Bit late to the party here sorry (have been traveling) but may also be nice to include a `CI` section, in particular with slowing down the autoupdates